firefox-ios/Client/Experiments/RecordedNimbusContext.swift

```diff
@@ -0,0 +1,185 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+import Glean
+import Shared
+
+import func MozillaAppServices.getCalculatedAttributes
+import func MozillaAppServices.getLocaleTag
+import struct MozillaAppServices.JsonObject
+import protocol MozillaAppServices.RecordedContext
+import MozillaRustComponents
+
+private extension Double? {
+    func toInt64() -> Int64? {
+        return self == nil ? nil : Int64(self!)
+    }
+}
+
+private extension Int32? {
+    func toInt64() -> Int64? {
+        return self == nil ? nil : Int64(self!)
+    }
+}
+
+/**
+ * The following constants are string constants of the keys that appear in the [EVENT_QUERIES] map.
+ */
+let DAYS_OPENED_IN_LAST_28: String = "days_opened_in_last_28"
+
+/**
+ * [EVENT_QUERIES] is a map of keys to Nimbus SDK EventStore queries.
+ */
+let EVENT_QUERIES = [
+    DAYS_OPENED_IN_LAST_28: "'events.app_opened'|eventCountNonZero('Days', 28, 0)",
+]
```

Comment:
It is generally not the best practice in Swift to have globals. Is there a specific reason this needs to be a global variable and can't be, for example, a static variable on RecordedNimbusContext?

Reply:
Nah we can add it as a static! I'll adjust that